### PR TITLE
Allow to install local package for testing

### PR DESCRIPTION
**Pull request by kenhys**

### CODE REVIEW

File: winevtlog_bench/ansible/windows.yml [hunk 1]
```diff
@@ -4,11 +4,22 @@
     td_agent_version: "3.8.0-0"
     eventlog_bencher_version: "v0.4.1.0"
   tasks:
+    - name: Copy local modified version for testing
+      win_copy:
+        src: ./config/td-agent-4.0.50-x64.msi
```
Comment (cosmo0920) [inline winevtlog_bench/ansible/windows.yml]: Hardcoded version is not desired on this repository.
Could you use a variable which is defined in `vars` and use it?

**Review by cosmo0920 — CHANGES_REQUESTED** [review]
I'd commented for this changes but , ideally, we should use separated task file for local installation.

File: winevtlog_bench/ansible/windows.yml [hunk 2]
```diff
@@ -4,11 +4,22 @@
...
+        dest: C:\td-agent.msi
+      when: local_package == "yes"
+    - name: Install local td-agent for testing
+      win_package:
+        path="c:\\td-agent.msi"
+        arguments="/q"
+      when: local_package == "yes"
     - name: Install td-agent
       win_package:
         path="http://packages.treasuredata.com.s3.amazonaws.com/3/windows/td-agent-{{ td_agent_version }}-x64.msi"
```
Comment (cosmo0920) [inline winevtlog_bench/ansible/windows.yml:19]: off topic: 

path="http://packages.treasuredata.com.s3.amazonaws.com/3/windows/td-agent-{{ td_agent_version }}-x64.msi"

**3** should be also configurable...

File: winevtlog_bench/ansible/windows.yml (hunk 1 above)
Comment (kenhys) [reply winevtlog_bench/ansible/windows.yml]: Yes, it is better to do so.

File: winevtlog_bench/ansible/windows.yml (hunk 2 above)
Comment (kenhys) [reply winevtlog_bench/ansible/windows.yml:19]: It may be better to handle another PR.

Comment (cosmo0920) [reply winevtlog_bench/ansible/windows.yml:19]: Sure.

**Review by cosmo0920 — APPROVED** [review]

**Comment by cosmo0920** [comment]
I'm waiting for CI results.

**Pull request merged**